tests/baselines/reference/contextualTypeOnYield2.types

```diff
@@ -5,7 +5,7 @@ type OrGen = () => (number | Generator<string, (arg: number) => void, undefined>
 
 const g: OrGen = function* () {
 >g : OrGen
->function* () {    return (num) => console.log(num);} : () => Generator<never, (num: number) => void, unknown>
+>function* () {    return (num) => console.log(num);} : () => Generator<never, (num: number) => void, undefined>

```

Comment:
All 3 affected baselines are directly related to the change and they are improvements/fixes